mcp-server/README.md

```diff
@@ -0,0 +1,107 @@
+# Wren MCP Server
+
+The **Wren MCP Server** is a **Model Context Protocol (MCP) server** that provides tools for interacting with **Wren Engine** to facilitate AI agent integration.
+
+## Requirements
+
+Before setting up the Wren MCP Server, ensure you have the following dependency installed:
+
+- **[uv](https://docs.astral.sh/uv/getting-started/installation/#installing-uv)** - A fast and efficient Python package manager.
+
+## Environment Variables
+
+The server requires the following environment variables to be set:
+
+| Variable | Description |
+|----------|------------|
+| `WREN_URL` | The URL of the **Wren Ibis server**. |
+| `CONNECTION_INFO_FILE` | The path to the **required connection info file**. |
+| `MDL_PATH` | The path to the **MDL file**. |
```

Comment:
Additionally, we should note that the mdl is not the same as a standard engine mdl. it need `datasource` field.

Reply:
Thanks for pointing this out. I added a section to mention it. By the way, `dataSource` is actually the standard MDL field.